### PR TITLE
Add section on "Best Practices for Phys Data Collection"

### LINKED ISSUE #99: Phys Data Best Practices Documentation
Phys data best practices documentation outline for OHBM abstract [here](https://docs.google.com/document/d/1Lv7jeTk3XnZP9GbNUI7cHs8ZoOuT1NxWrlQIPg6n9Y0/edit?usp=sharing).

Feel free to edit as you see fit.

### PR DESCRIPTION
**Pull request by AyyagariA**
Closes #99 addresses #12

## Proposed Changes

  - Merging the changes and updates for the documentation on best practices for collecting physiological data
  -
  -

### CODE REVIEW

**Comment by smoia** [comment]
Hello there, any update on this?

**Comment by AyyagariA** [comment]
Ready for review!

**Comment by smoia** [comment]
Great!

Can I ask @RayStick, @BrightMG, and/or @CesarCaballeroGaudes to review this PR?

**Comment by smoia** [comment]
I think that @BrightMG is looking into this with a PR to @AyyagariA's branch.
@BrightMG , can you let us know here when it's ready to be reviewed?

**Comment by BrightMG** [comment]
@AyyagariA's PR now contains my updated version of the best practices doc. I'd appreciate another person reviewing this now

**Comment by RayStick** [comment]
I can have a read through soon

**Comment by AyyagariA** [comment]
Thanks Molly for your feedback and guidance! @kristinazvolanek and I will be adding references and making a few edits this week.

**Comment by RayStick** [comment]
> I can have a read through soon, if that's welcome?
> I can review tomorrow morning (CST)

I don't think I'll get to it today! @AyyagariA shall I wait until you've looked over it again, or shall read when I can (probably tomorrow)?

**Comment by AyyagariA** [comment]
> > I can have a read through soon, if that's welcome?
> > I can review tomorrow morning (CST)
> 
> I don't think I'll get to it today! @AyyagariA shall I wait until you've looked over it again, or shall read when I can (probably tomorrow)?

There aren't any major content changes for right now, so just read when you get a chance and we will let you know if there are updates! Thank you.

File: docs/bestpractice.rst [hunk 1]
```diff
@@ -47,3 +50,60 @@ What to do with physiological data once it has been collected?
...
... dioxide levels (CO2), which is a known vasodilator and can drive large variability in blood flow and the BOLD signal. We can also measure oxygen (O2) levels; O2 only has a mild vasoconstictive effect on the cerebrovasculature, but oxygen levels can directly influence BOLD signal contrast. These two blood gases are typically strongly anticorrelated with eachother in most scans, but can also be manipulated independently and influence the fMRI signals through distinct mechanisms. Best practice would be to record both. Although the most accurate recordings of blood gas levels would be achieved through arterial sampling, this is not recommended for most imaging experiments. Instead, the concentrations of CO2 and O2 in arterial blood can be approximated by the partial pressure of each gas at the end of an exhalation, or the end-tidal partial pressure (commonly abbreviated as PETCO2 and PETO2). The person being scanned wears a nasal cannula (soft plastic tube that rests just below the nostrils) or face mask that is connected to a gas analyzer in the control room. The resulting data shows the fluctuations in CO2 and O2 across every breath; an algorithm must extract the "end-tidal" values.
```
Comment (RayStick) [inline docs/bestpractice.rst:78]: Remove ":"

Comment (RayStick) [reply docs/bestpractice.rst:78]: It says 'oxygen' after introducing the abbreviation 'O2'

File: docs/bestpractice.rst [hunk 2]
```diff
@@ -47,3 +50,60 @@ What to do with physiological data once it has been collected?
...
+
+
+What equipment is needed?
+#########################
+
+Peripheral devices:
+
+- finger photoplethysmograph (pulse-oximeter)
+- respiratory belt
+- disposable nasal cannula (or face mask) and long sample line to connect to control room
```
Comment (RayStick) [inline docs/bestpractice.rst:88]: "long sample line to connect from the scan room to the control room"

File: docs/bestpractice.rst [hunk 3]
```diff
@@ -47,3 +50,60 @@ What to do with physiological data once it has been collected?
...
+
+Some peripherals can be passed through a void in the penetration panel from the control room to the scan room (e.g., gas sampling line); others must be plugged into the penetration panel for noise filtering (e.g., some pulse sensors). Devices native to the MRI scanner may communicate wirelessly with the scanner. When adding non-native peripheral devices to the scanner environment, we recommend that you check that you are not bringing any outside noise into the scan room or bringing too much scanner noise into the physiological recordings. It may be necessary to develop additional devices or mechanisms to shield these connections.
```
Comment (RayStick) [inline docs/bestpractice.rst:90]: "Some peripheral devices"

**Comment by RayStick** [comment]
I gave it a read - reads well and good to go in my opinion (after references have been added, and formatting checked, etc.). There will always be more to say on this topic but I think this is a good concise introductory overview! 

There are a few very minor wording comments I made.

**Comment by AyyagariA** [comment]
@smoia This is ready to go! Just pushed the final touches.

**Comment by smoia** [comment]
@RayStick , if you approve, merge it in!

**Comment by RayStick** [comment]
> @RayStick , if you approve, merge it in!
Sure! (Was just reading the contributor file to remind myself whether I was allowed!)

**Pull request merged**